[BUG] `get-shim-versions-from-dist` workflow failing in CI

**Describe the bug**
Multiple PRs are failing with:

```
+++ mvn -B help:evaluate -q -pl dist -Psnapshots -Dexpression=included_buildvers -DforceStdout
Error: K_SHIM_VERSIONS_STR='[ERROR] No plugin found for prefix '\''help'\'' in the current project and in the plugin groups [org.apache.maven.plugins, org.codehaus.mojo] available from the repositories [local (/home/runner/.m2/repository), central (https://repo.maven.apache.org/maven2)] -> [Help 1]
```

**Steps/Code to reproduce bug**
See PRs such as https://github.com/NVIDIA/spark-rapids/pull/8152 or https://github.com/NVIDIA/spark-rapids/pull/8102

**Expected behavior**
Should not fail

**Environment details (please complete the following information)**
GitHub actions

**Additional context**

--- a/pom.xml
+++ b/pom.xml
@@ -1282,5 +1282,23 @@
                 <enabled>true</enabled>
             </snapshots>
         </repository>
+        <repository>
+            <id>central</id>
+            <name>Maven Repository Switchboard</name>
+            <layout>default</layout>
+            <url>https://repo1.maven.org/maven2</url>
+            <snapshots>
+                <enabled>false</enabled>
+            </snapshots>
+        </repository>
     </repositories>
+    <pluginRepositories>
+        <pluginRepository>
+            <id>central</id>
+            <url>https://repo1.maven.org/maven2</url>
+            <snapshots>
+                <enabled>false</enabled>
+            </snapshots>
+        </pluginRepository>
+    </pluginRepositories>
 </project>

